gradle.properties

```diff
@@ -3,4 +3,4 @@ org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.caching=true
 
-BUILD_VERSION_PREFIX=1.0.1
+BUILD_VERSION_PREFIX=1.1.0
```

Comment:
Updating the acapy version is meaningful for internal features and stability, I suggest to bump the version.
With the current change in start_wallet_from_seed.sh this is not a breaking change, so the minor change is true.